authoring-service/src/test/kotlin/com/devotion/authoring/service/DefaultSurveyAuthoringServiceTest.kt

```diff
@@ -39,9 +37,9 @@ class DefaultSurveyAuthoringServiceTest {
     private lateinit var service: DefaultSurveyAuthoringService
 
     var survey_0 = Optional.of(Survey(published = true))
-    var survey_1 = Optional.of(Survey(published = false, questions = DefaultSurveyAuthoringServiceTest.getQuestions(4)))
+    var survey_1 = Optional.of(Survey(published = false).apply { questions.addAll(DefaultSurveyAuthoringServiceTest.getQuestions(4))})
```

Comment:
Can you call these guys `survey0`, `survey1` to be java style? Or even something meaningful, since it's not clear by the name what purpose they serve

Reply:
they have meaning like survey with id 1,2...